api/get_commitment_proof_test.go

```diff
@@ -102,7 +102,8 @@ func (s *GetCommitmentProofTestSuite) TestGetCommitmentProof_TransferType() {
 	s.addStateLeaf()
 
 	transfer := testutils.MakeTransfer(1, 2, 0, 50)
-	transfer.CommitmentID = &s.txCommitment.ID
+	slot := models.MakeCommitmentSlot(s.txCommitment.ID, 0)
+	transfer.CommitmentSlot = &slot
```

Comment:
How about creating `NewCommitmentSlot` function returning a pointer and making such assignments one-liners?

Reply:
Great point!